python/ola/PidStore.py

```diff
@@ -892,7 +892,7 @@ def GetDescription(self, indent=0):
       names.append('<%s>' % atom.name)
       output.append(atom.GetDescription(indent=2))
 
-    return ' '.join(names) + ':\n' + '\n'.join(output)
+    return ' '.join(names), '\n'.join(output)
```

Comment:
Thanks, would you mind correcting the comment docs in the base class to explain this.

Reply:
Done.